Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -1096,6 +1096,23 @@ private static void RunInitializeInPlayer()
 #if UNITY_EDITOR
         private static InputSystemObject s_SystemObject;
 
+        private static void CheckCurrentPlayerSettingForInputSystem()
+        {
+            PlayerSettings[] array = Resources.FindObjectsOfTypeAll<PlayerSettings>();
+            var playerSettings = array.Length > 0 ? array[0] : null;
+            if (playerSettings)
+            {
+                var s_PlayerSettings = new SerializedObject(playerSettings);
+
+                // Only warn the user once.  Save the warning flag across domain reloads.
+                if (s_PlayerSettings.FindProperty("enableNativePlatformBackendsForNewInputSystem").boolValue == false && !s_Manager.oldInputWarnUserFlag)
+                {
+                     EditorUtility.DisplayDialog("Input Manager Active", "You currently have the input system package installed, however it is not active due to the Active Input Handling player setting set to Input Manager.", "OK");
```

Comment:
This should both explain the situation to the user ("This project is using the new input system package but the native platform backends for the new input system are not enabled in the player settings. This means that no input from native devices will come through.") and also give the user a choice for what to do ("Do you want to enable the backends. Doing so requires a restart of the editor."; choice "Enable New&Old Backends", "Enable New Backends", "Do Nothing").